jhsingle_native_proxy/main.py

```diff
@@ -1,7 +1,8 @@
 from tornado.httpserver import HTTPServer
 from tornado import ioloop
 from tornado.web import Application, RequestHandler, RedirectHandler
-from tornado.log import app_log
+import tornado.log
```

Comment:
Do you need to import the whole thing on its own?

Reply:
removed tornado.log from import